src/controllers/userController.js

```diff
@@ -19,5 +19,30 @@ const User = {
     const data = { ...userData, token };
     return sendResult(res, 201, 'Account created successfully', data);
   },
+  async login(req, res) {
+    const { email, password } = req.body;
+    const userInfo = await db.Users.findOne({
+      where: {
+        email,
+      },
+    });
+    if (!userInfo) {
+      return sendResult(res, 401, 'The email and/or password is invalid');
+    }
+
+    const comfirmPass = helpers.comparePassword(password, userInfo.password);
+    if (comfirmPass) {
+      const token = helpers.createToken(userInfo.id, userInfo.email, false);
```

Comment:
the third parameter of `createToken()` is `isverified`. instead of passing false, you should pass `userInfo.isverified`